### PR TITLE
Drawer styles updates

### PR DESCRIPTION
**Pull request by Kritik-J**
Fixes Issue - https://github.com/coral-xyz/backpack/issues/2760

Problem - Drawer polish (light mode only) #2760

Solution - Changed backdropProps style opacity from 0.8 to 0.2 and approveTransactionCloseBackground from #C2C4CC to #F0F0F2

### CODE REVIEW

File: packages/app-extension/src/components/common/ApproveTransactionDrawer.tsx [hunk 1]
```diff
@@ -44,7 +44,7 @@ export const ApproveTransactionDrawer: React.FC<{
       setOpenDrawer={setOpenDrawer}
       backdropProps={{
         style: {
-          opacity: 0.8,
+          opacity: 0.2,
```
Comment (armaniferrante) [inline packages/app-extension/src/components/common/ApproveTransactionDrawer.tsx:47]: there's a lot more places where thsi is used

File: packages/themes/src/index.tsx [hunk 2]
```diff
@@ -329,7 +329,7 @@ const LIGHT_COLORS: CustomColors = {
   swapTokensButton: "#FFFFFF",
   icon: "#8F929E",
   approveTransactionTableBackground: LIGHT_BACKGROUND_COLOR_1,
-  approveTransactionCloseBackground: "#C2C4CC",
+  approveTransactionCloseBackground: "#F0F0F2",
```
Comment (armaniferrante) [inline packages/themes/src/index.tsx:332]: The original issue says to change the border. Does this do that?

**Comment by armaniferrante** [comment]
Please share images of these changes

File: packages/themes/src/index.tsx (hunk 2 above)
Comment (Kritik-J) [reply packages/themes/src/index.tsx:332]: Yes, as I did not find any border property here. Instead, I discovered that the close button in light mode has a backgroundColor of "#FFFFFF" and a width and height of 38 px, which is wrapped inside a div with a backgroundColor of #C2C4CC and a width and height of 44 px.

File: packages/app-extension/src/components/common/ApproveTransactionDrawer.tsx (hunk 1 above)
Comment (Kritik-J) [reply packages/app-extension/src/components/common/ApproveTransactionDrawer.tsx:47]: Thanks for pointing this out; I will fix others as well.

File: packages/themes/src/index.tsx (hunk 2 above)
Comment (Kritik-J) [reply packages/themes/src/index.tsx:332]: This is the output -

drawer

**Comment by Kritik-J** [comment]
@armaniferrante These are some images of the changes

drawer-1

drawer-2

drawer-3

drawer-4

drawer-5

drawer-6

**Pull request closed without merging**